src/cart/crt_ctl.c

```diff
@@ -75,6 +75,135 @@ static int verify_ctl_in_args(struct crt_ctl_ep_ls_in *in_args)
 	return rc;
 }
 
+int
+crt_ctl_fill_buffer_cb(d_list_t *rlink, void *arg)
+{
+	struct crt_uri_item	*ui;
+	int			*idx;
+	char			*buf;
+	struct crt_uri_cache	*uri_cache = arg;
+	int			 i;
+	int			 uri_size;
+	int			 rc = 0;
+
+	D_ASSERT(rlink != NULL);
+	D_ASSERT(arg != NULL);
+
+	ui = crt_ui_link2ptr(rlink);
+
+	D_MUTEX_LOCK(&ui->ui_mutex);
+	for (i = 0; i < CRT_SRV_CONTEXT_NUM; i++) {
+		if (ui->ui_uri[i] != NULL) {
```

Comment:
instead of doing if (ui->ui_uri[i] != NULL) { } its better to do something like
if (ui->ui_uri[i] == NULL)
    continue;

this eliminates need for extra indentation